src/components/Base/Hero.stories.js

```diff
@@ -0,0 +1,89 @@
+import { storiesOf } from "@storybook/vue";
+import { VApp, VContent } from "vuetify/lib"; // <-- add the import
+
+import Hero from "./Hero.vue";
+import yoga2x from '../../assets/yoga@2x.png';
+
+// Decorator for adding Storybook styles around the component to be rendered
+const appDecorator = () => {
+  return {
+    components: { VApp, VContent },
+    template: `
+      <v-app>
+        <div style="padding: 20px; width: 100%; height: 100%;">
+          <v-content>
+            <story />
+          </v-content>
+        </div>
+      </v-app>
+    `
+  };
+};
+
+storiesOf("Hero", module)
+  .addDecorator(appDecorator)
+  .add("default", () => ({
+    components: { Hero },
+    data: function() {
```

Comment:
Optionally using ES6 syntax:

```
data: () => ({
  image: yoga2x
}),
```

*Special note*: this ES6 syntax works fine for `data` but not for other lifecycle hooks. Vue does some scoped binding of `this` and as such requires standard function declaration, ie. `created: function() {}`. Since this is the case, using standard function declaration for `data` (as you have done) provides consistency.